src/analyz/ty_json.rs

```diff
@@ -397,15 +397,17 @@ impl<'tcx> ToJson<'tcx> for ty::Instance<'tcx> {
                 // with the cases in rustc's `build_clone_shim` function (see
                 // https://doc.rust-lang.org/1.86.0/nightly-rustc/src/rustc_mir_transform/shim.rs.html#432-455).
                 //
-                // TODO(#196): We are currently missing two cases for
-                // TyKind::Coroutine and TyKind::CoroutineClosure. We should
-                // add these cases, taking inspiration from how
+                // TODO(#196): We are currently missing a case for
+                // TyKind::Coroutine. We should
+                // add this cases, taking inspiration from how
```

Comment:
```suggestion
                // add this case, taking inspiration from how
```

Reply:
Fixed in bb9ba85b53d6da6ceffd0ba31f174b01820e0b67